src/main/resources/application.yaml

```diff
@@ -29,8 +29,8 @@ madie:
       cql:
         uri: /cql-libraries/cql
   translatorVersion:
-      currentVersion: ${CURRENT_TRANSLATOR_VERSION:3.14.0}
-      mostRecentVersion: ${MOST_RECENT_TRANSLATOR_VERSION:3.14.0}
+      currentVersion: ${CURRENT_TRANSLATOR_VERSION:3.24.0}
+      mostRecentVersion: ${MOST_RECENT_TRANSLATOR_VERSION:3.24.0}
```

Comment:
should this be same as well https://github.com/MeasureAuthoringTool/madie-fhir-elm-translator/pull/56/files?